# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: architecture/sparc64/pkg_gcc.diff
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gcc-3.4.3/gcc/config/sparc/linux64.h.oir	2005-06-08 09:17:32.000000000 +0200
+++ gcc-3.4.3/gcc/config/sparc/linux64.h	2005-06-08 09:26:59.000000000 +0200
@@ -45,11 +45,13 @@
 /* A 64 bit v9 compiler with stack-bias,
    in a Medium/Low code model environment.  */
 
+#ifdef SPARC64_BECAME_ROCK_SOLID_AND_X_USABLE_RENER
 #undef TARGET_DEFAULT
 #define TARGET_DEFAULT \
   (MASK_V9 + MASK_PTR64 + MASK_64BIT /* + MASK_HARD_QUAD */ \
    + MASK_STACK_BIAS + MASK_APP_REGS + MASK_FPU + MASK_LONG_DOUBLE_128)
 #endif
+#endif
 
 #undef ASM_CPU_DEFAULT_SPEC
 #define ASM_CPU_DEFAULT_SPEC "-Av9a"
